modules/oc-mirror-creating-image-set-config.adoc

```diff
@@ -68,6 +68,11 @@ mirror:
 <9> Specify any additional images to include in image set.
 +
 See _Image set configuration parameters_ for the full list of parameters and _Image set configuration examples_ for various mirroring use cases.
++
+[NOTE]
+====
+The `graph: true` field also generates the `ubi` image along with other mirrored images.
+====
```

Comment:
Is it at all possible to insert this note into the `<5>` (L64)?
Just a suggestion:
```suggestion
+
<5> Add `graph: true` to build and push the graph-data image to the mirror registry. The graph-data image is required to create OpenShift Update Service (OSUS). The image is based on `ubi-micro`. The `graph: true` field mirrors both the graph-data image and the `ubi-micro` image. It  also generates the `UpdateService` custom resource manifest. The `oc` command-line interface (CLI) can use the `UpdateService` custom resource manifest to create OSUS. For more information, see _About the OpenShift Update Service_.
```

Reply:
Yeah, I can move the note up.